src/Screens/about/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function About() {
```

Comment:
I would suggest to convert them all to ES6 functions. (Since they're faster and better practice)

Reply:
done